tools/bpf2c/bpf2c.cpp

```diff
@@ -51,6 +72,9 @@ main(int argc, char** argv)
                 {"--dll",
                  {"Generate code for a Windows DLL",
                   [&](std::vector<std::string>::iterator&) { type = output_type::UserPE; }}},
+                {"--verify",
+                 {"Validate code using verifier",
```

Comment:
When would not want to specify this?   Should we default to being more secure and require specifying noverifyif you don't want it for some reason?